docs/teem.rst

```diff
@@ -0,0 +1,39 @@
+TEEM – F5 ACI ServiceCenter data usage collection
+=================================================
+
+The purpose of this feature is to provide anonymous usage analytics to F5 by using the F5 TEEM infrastructure. As a part of this feature usage data is collected once per day regarding F5 ACI ServiceCenter usage, and a report is sent to the TEEM data collection servers, where the usage data is further analyzed. 
+
+Note: This does not include any customer specific data such as BIG-IP device IPs or hostnames. 
```

Comment:
consider bolding "does not" or changing "Note:"  to "Important:" or even an important callout box in rst, ie:

.. IMPORTANT:: This does not include any...